src/bundle/Resources/public/js/scripts/admin.limitation.pick.js

```diff
@@ -1,13 +1,162 @@
+                method: 'POST',
+                headers: {
+                    Accept: 'application/vnd.ez.api.View+json; version=1.1',
+                    'Content-Type': 'application/vnd.ez.api.ViewInput+json; version=1.1',
+                    'X-Requested-With': 'XMLHttpRequest',
+                },
+                content: JSON.stringify({
+                    ViewInput: {
+                        identifier: `udw-locations-by-path-string-${pathArray.join('-')}`,
+                        public: false,
+                        LocationQuery: {
+                            Criteria: {},
+                            FacetBuilders: {},
+                            SortClauses: { SectionIdentifier: 'ascending' },
+                            Filter: { LocationIdCriterion: pathArray.join(',') },
+                            limit: 50,
+                            offset: 0,
+                        },
+                    },
+                }),
+            };
+
+            return operations;
+        }, {});
+    const removeRootLocation = (pathArray) => pathArray.slice(1);
+    const pathStringToPathArray = (pathString) => pathString.slice(1, -1).split('/');
+    const buildContentBreadcrumbs = (viewData) => {
+        const searchHitList = viewData.View.Result.searchHits.searchHit;
+
+        return searchHitList.map((searchHit) => searchHit.value.Location.ContentInfo.Content.Name).join(' / ');
+    };
+    const addLocationsToInput = (limitationBtn, newlySelectedItems) => {
+        const input = doc.querySelector(limitationBtn.dataset.locationInputSelector);
+        const alreadySelectedLocationsIds = input.value ? input.value.split(IDS_SEPARATOR) : [];
+        const newlySelectedLocationsIds = newlySelectedItems.map((item) => item.id);
+        const allSelectedLocationsIds = [...alreadySelectedLocationsIds, ...newlySelectedLocationsIds].join(IDS_SEPARATOR);
+
+        input.value = allSelectedLocationsIds;
+    };
+    const removeLocationFromInput = (limitationBtn, removedLocationId) => {
+        const input = doc.querySelector(limitationBtn.dataset.locationInputSelector);
+        const locationsIdsWithoutRemoved = input.value.split(IDS_SEPARATOR).filter((locationId) => locationId !== removedLocationId);
+
+        input.value = locationsIdsWithoutRemoved.join(IDS_SEPARATOR);
+    };
+    const addLocationsTags = (limitationBtn, newlySelectedItems) => {
+        const tagsList = doc.querySelector(limitationBtn.dataset.selectedLocationListSelector);
+        const tagTemplate = limitationBtn.dataset.valueTemplate;
+        const fragment = doc.createDocumentFragment();
+
+        newlySelectedItems.forEach((location) => {
+            const contentName = location.ContentInfo.Content.Name;
+            const locationId = location.id;
+            const container = doc.createElement('ul');
+            const renderedItem = tagTemplate.replace('{{ contentBreadcrumbs }}', contentName).replace('{{ locationId }}', locationId);
```

Comment:
You replace `contentBreadcrumbs` with `contentName` should be breadcrumbs as well?

Reply:
Ohhh, I forgot to delete this. We can generate in twig template with `'{{ contentBreadcrumbs }}'` set to empty string, because we are always fetching names from server, so this contentName will never be shown. I will drop this replacement and adapt twig.